test/test_cli.rb

```diff
@@ -46,6 +46,49 @@ def teardown
     @ready.close
   end
 
+  def check_single_stats(body, check_puma_stats = true)
+    http_hash = JSON.parse body.split("\n", 2).last
```

Comment:
i might do this in two lines with variable names that indicate what the split and last achieve

Reply:
Actually, that isn't needed.  My mistake.  Removed.

The tests in `test_integration_pumactl.rb` use `Puma::ControlCLI` which outputs the JSON string to `@stdout`, but prefaces it with a line "Command stats sent success".

`test_cli.rb` sends a simple request, and only the JSON string is returned.  I was thinking it also had the line that needed to be stripped...

Reply:
nice

is it valuable to also test the value of `first` as well?

Reply:
There's a few lines in `test_integration_pumactl.rb` that check it. The code in `puma/control_cli.rb` is:

https://github.com/puma/puma/blob/36c7307607432416d9a58eef317757da1eceb95a/lib/puma/control_cli.rb#L225-L226